core/templates/dev/head/pages/base.html

```diff
@@ -103,16 +103,15 @@
         <div ng-if="isBackgroundMaskActive()" class="ng-cloak oppia-background-mask">
         </div>
 
-        <div class="oppia-base-container"
-             ng-class="{'oppia-sidebar-menu-open': isSidebarShown(), 'oppia-sidebar-menu-closed': !isSidebarShown()}"
-             ng-swipe-left="closeSidebarOnSwipe()" ng-swipe-disable-mouse="false">
+        <div class="oppia-base-container" ng-class="{'oppia-sidebar-menu-open': isSidebarShown(),
```

Comment:
Probably better to keep the line breaks as they were previously, so that you do not need to split an attr value across multiple lines.

Reply:
i agree . updated PTAL Thanks!